File: Basic Coding Problems/01 Fundamentals/016 Write a program to enter marks of five subjects and calculate total, average and percentage/C/marks.c
#include <stdio.h>

int main() {
    int subject1, subject2, subject3, subject4, subject5;
    int total;
    float average;
    float percentage;

    printf("Enter the marks of the subjects: ");
    scanf("%d %d %d %d %d", &subject1, &subject2, &subject3, &subject4, &subject5);

    total = subject1 + subject2 + subject3 + subject4 + subject5;
    average = total / 5.0;
    percentage = (total / 500.0) * 100;

    printf("The total marks of the subjects is: %d\n", total);
    printf("The average marks of the subjects is: %.2f\n", average);
    printf("The percentage of the total marks of the subjects is: %.2f%%\n", percentage);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/016 Write a program to enter marks of five subjects and calculate total, average and percentage/Java/marks.java
import java.util.Scanner;

public class marks {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the marks of the subjects: ");
        int subject1 = scanner.nextInt();
        int subject2 = scanner.nextInt();
        int subject3 = scanner.nextInt();
        int subject4 = scanner.nextInt();
        int subject5 = scanner.nextInt();

        int total = subject1 + subject2 + subject3 + subject4 + subject5;
        float average = total / 5.0f;
        float percentage = (total / 500.0f) * 100;

        System.out.println("The total marks of the subjects is: " + total);
        System.out.println("The average marks of the subjects is: " + average);
        System.out.println("The percentage of the total marks of the subjects is: " + percentage + "%");
    }
}